File: Assignment 4/Question3.c
#include <stdio.h>
void DisplayNonFactor(int iNo)
{
    if(iNo <0)
    {
        iNo = -iNo ;
    }
    int iCnt=0;
    for ( iCnt=1 ; iCnt<iNo ; iCnt++)
    {
        if(iNo % iCnt != 0)
        {
                printf("%d\t",iCnt);       
        }
    }
}

int main()
{
    int iValue = 0 ;
    printf("Enter Number : ");
    scanf("%d",&iValue);
    DisplayNonFactor(iValue);
   
   

    return 0 ;
}

File: Assignment 18/Question3.java
import java.util.Scanner;

class Logic
{
    
    public void PrintOdd(int iNo) 

    {
        int iCnt = 0;
        
        for (iCnt = 1 ; iCnt<=iNo ; iCnt= iCnt+2)
        {
            System.out.println(iCnt);     
        }   
       
    }  
}          
class Question3
{
    public static void main(String[] args) 
    {
        
        Scanner sobj = new Scanner(System.in);
        int iValue1 = 0 ;
       
        System.out.println("Enter your Number :");
        iValue1 = sobj.nextInt();

        Logic lobj = new Logic();

        lobj.PrintOdd(iValue1);
    }    
}
